scripts/prettier/index.js

```diff
@@ -89,7 +91,10 @@ Object.keys(config).forEach(key => {
   args.push(`--${shouldWrite ? 'write' : 'l'}`);
 
   try {
-    exec(prettierCmd, [...args, ...files]);
+    const invalidFiles = exec(prettierCmd, [...args, ...files]);
+    if (!shouldWrite && invalidFiles) {
+      throw Error(invalidFiles);
+    }
```

Comment:
Can we store invalidFiles in a local var and read it in the catch block instead of throwing a generic exception? It's not great that we'd end up swallowing arbitrary exceptions and/or putting error messages where the filenames would go in the error message printed. (Making a new exception class specifically for this that you check for could also work, but I don't think it's necessary here.)